backend/inspirehep/assign/views.py

```diff
@@ -97,6 +99,24 @@ def author_assign_view():
     return jsonify({"message": "Success"}), 200
 
 
+@blueprint.route("create-ticket-for-claimed-papers", methods=["POST"])
+@login_required_with_roles()
+def author_assign_view_claimed():
+    body = request.get_json()
+    to_author_recid = body.get("to_author_recid")
+    from_author_recid = body["from_author_recid"]
+    claimed_literature_recids = body["papers_ids_already_claimed"]
+    not_allowed_to_be_claimed_literature_recids = body["papers_ids_not_matching_name"]
+    create_rt_ticket_for_claiming_action(
+        from_author_recid,
+        to_author_recid,
+        claimed_literature_recids,
+        not_allowed_to_be_claimed_literature_recids,
+    )
+    db.session.commit()
+    return jsonify({"message": "Success"}), 200
```

Comment:
I have two comments about this:
1. Why we don't have it as part of the claiming? A request might fail.
2. I will make the `create_rt_ticket_for_claiming_action` async since it does some querying and takes time especially if the user select for example 100 papers.

Reply:
I thought that I'll make another endpoint cause we might do 2 actions for this type of claiming (if you have claimed & unclaimed in your selection) and in case of claimedPapers we need to split papersIds into papers that are claimed and the ones with wrong name so the current args in the default assign view don't match the new ones and the logic becomes more complex. Also we don't assign an author in that case, we only create a ticket.

Reply:
I see, I would make the `create_rt_ticket_for_claiming_action` async not to wait the request